### DISCUSSION TITLE
How to handle this error in the docs: "When I sign in with another account with the same email address, why are accounts not linked automatically?"

### DISCUSSION THREAD

**Discussion by wakywayne** [nextauthjs/next-auth#5795] [Ideas]
### Question 💬

I understand the problem, but when a user who has already authenticated with google for example tries to authenticate with facebook I just get the following error in the URL: 
http://localhost:3000/auth/signin?callbackUrl=http%3A%2F%2Flocalhost%3A3000%2Fauth%2Fsignin%3FcallbackUrl%3Dhttp%253A%252F%252Flocalhost%253A3000%252Fauth%252Fsignin%253FcallbackUrl%253D%252F%26error%3DOAuthAccountNotLinked%23_%3D_&error=OAuthAccountNotLinked#_=_

I was wondering if there is a better way to handle this error and if there is a way to specifically display a message if this issue happens

Here is the error part: error=OAuthAccountNotLinked#_=_

and here is the link to the docs where it explains this issue: https://next-auth.js.org/faq#:~:text=When%20I%20sign%20in%20with%20another%20account%20with%20the%20same%20email%20address%2C%20why%20are%20accounts%20not%20linked%20automatically%3F

### How to reproduce ☕️

any Oauth application that has more then one external providers can replicate:

1. Authenticate a new user with a provider
2. Log user out and have them try to log back in with a different provider
3. You should get the same url error

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
The error is correct, improving the error message though is a welcome change. If you open a PR, I'm happy to assist.

**Reply by wakywayne** [reply]
Two questions one, how do I open a PR and two? Is there a better way that we can handle this on this client side? Right now it just leaves the login form and doesn't reroute. I'd like to display a pop-up or something.
